_styleguide.html

```diff
@@ -1108,8 +1109,8 @@
 <footer class="footer">
 </footer>
 
-    <script src="static/dist/bower_components/jquery/dist/jquery.min.js"></script>
-    <!--[if lt IE 9 ]> 
+    <script src="static/dist/js/vendor.min.js"></script>
```

Comment:
Does minifying and concatenating all third party js files into one `vendor.js` file make the assumption that all third party dependencies will be required on every page? as this is often not the case.